multisig/proposals/arbitrumone/vip-013/index.ts

```diff
@@ -0,0 +1,232 @@
+      params: [
+        [
+          weETH,
+          [
+            weETH_ONE_JUMP_ORACLE,
+            "0x0000000000000000000000000000000000000000",
+            "0x0000000000000000000000000000000000000000",
+          ],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[weETH, CHAINLINK_weETH_FEED, STALE_PERIOD_26H]],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [weETH, parseUnits("2", 18), arbitrumone.GUARDIAN],
+    },
+    {
+      target: weETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, 0],
+    },
+    {
+      target: weETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, parseUnits("2", 18)],
+    },
+    {
+      target: VweETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VweETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
+      target: VweETH,
+      signature: "setProtocolSeizeShare(uint256)",
+      params: [parseUnits("0.01", 18)],
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VweETH,
+          parseUnits("0.93", 18),
+          parseUnits("0.95", 18),
+          parseUnits("2", 18),
+          vTokenReceiver,
+          parseUnits("4600", 18),
+          parseUnits("2300", 18),
+        ],
+      ],
+    },
+    {
+      target: arbitrumone.VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [WETH, parseUnits("1.9678", 18), arbitrumone.GUARDIAN],
+    },
+    {
+      target: WETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, "0"],
+    },
+    {
+      target: WETH,
+      signature: "approve(address,uint256)",
+      params: [arbitrumone.POOL_REGISTRY, parseUnits("1.9678", 18)],
+    },
+    {
+      target: VWETH,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["86400"],
+    },
+    {
+      target: VWETH,
+      signature: "setProtocolShareReserve(address)",
+      params: [PSR],
+    },
+    {
+      target: VWETH,
+      signature: "setProtocolSeizeShare(uint256)",
+      params: [parseUnits("0.01", 18)],
+    },
+    {
+      target: arbitrumone.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          VWETH,
+          parseUnits("0.77", 18),
+          parseUnits("0.8", 18),
+          parseUnits("1.9678", 18),
+          vTokenReceiver,
+          parseUnits("14000", 18),
+          parseUnits("12500", 18),
+        ],
+      ],
+    },
```

Comment:
We need to configure the XVS rewards for the new markets according to the [proposal in the forum](https://community.venus.io/t/incentive-model-proposal-for-arbitrum-deployment/4408):

* wstETH: 2,550 XVS for 90 days - 100% suppliers (327932098765432 XVS/second)
* weETH: 2,550 XVS for 90 days - 100% suppliers (327932098765432 XVS/second)
* WETH: 10,200 XVS for 90 days - 30% suppliers / 70% borrowers (393518518518518 XVS/second / 918209876543209 XVS/second)

(90 days = 7776000 seconds)

The source of the XVS (15,300 XVS total) is the Venus Treasury on Arbitrum

Reply:
Added here, b3a8a13b826b0b56963f32dae522ec9886f9dba7